src/app/component/app-survey/survey-form/survey-form.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="main-survey flex items-center justify-center" *ngIf="surveyPopup.active && isTourDone && localStorageFlag">
+    <div class="surver-inner">
```

Comment:
Please do optional check for surveyPopup

Reply:
applied